planning/autoware_static_centerline_generator/README.md

```diff
@@ -46,16 +46,29 @@ The optimized centerline can be generated from the command line interface by des
 - `<input-osm-path>`
 - `<output-osm-path>` (not mandatory)
 - `<start-lanelet-id>`
+- `<start-pose>` (not mandatory)
 - `<end-lanelet-id>`
+- `<end-pose>` (not mandatory)
 - `<vehicle-model>`
+- `<goal-method>` (not mandatory, `path_generator` or `fixed_goal_planner` only)
```

Comment:
In my understanding, `path_generator`'s goal pose is also the fixed goal, so `fixed_goal_planner` is hard to understand.
How about `path_generator` and `behavior_path_planner`?